packages/client/src/forms/register/fieldMappings/birth/mutation/registration-mappings.ts

```diff
@@ -187,7 +187,7 @@ export const customFieldToQuestionnaireTransformer = (
   }
   transformedData.questionnaire.push({
     fieldId: field.customQuesstionMappingId,
-    value: value
+    value: String(value)
```

Comment:
GraphQL schema expects the value to be string. In case of TEL or NUMBER fields, the value will be of number type so converting them to string